modules/standard/IO.chpl

```diff
@@ -7018,6 +7043,105 @@ iter fileReader.lines(stripNewline = false) {
   this.unlock();
 }
 
+@chpldoc.nodoc
+iter fileReader.lines(
+  param tag: iterKind,
+  stripNewline = false,
+  targetLocales: [?tld] locale = [this._home,]
+): string
+  where tag == iterKind.standalone
+{
+  on this._home {
+    try! this.lock();
+
+    var f = chpl_fileFromReaderOrWriter(this);
+
+    const myStart = qio_channel_get_start_pos(this._channel_internal),
+          myEnd = qio_channel_get_end_pos(this._channel_internal),
+          myBounds = myStart..<(try! min(myEnd, f.size));
```

Comment:
`min` can throw? I didn't think `min` could throw. Do you know why `try!` is here?

Reply:
`f.size` can throw. I'll move the `try!` to make it clear.